dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -3510,13 +3541,77 @@ BlockInputStreamPtr Segment::getBitmapFilterInputStream(
 {
     sanitizeCheckReadRanges(__FUNCTION__, read_ranges, rowkey_range, log);
 
-    auto bitmap_filter = buildBitmapFilter(
+    BitmapFilterPtr bitmap_filter = nullptr;
+    if (executor && executor->column_range && executor->column_range->type != ColumnRangeType::Unsupported)
+    {
+        bool all_dmfile_packs_skipped
+            = std::any_of(pack_filter_results.begin(), pack_filter_results.end(), [](const auto & res) {
+                  return res->countUsePack() == 0;
+              });
```

Comment:
Should it be `std::all_of` instead of `std::any_of`?

Reply:
Yes..., fixed.